src/Raven.Server/Documents/Handlers/Processors/Queries/DatabaseQueriesHandlerProcessorForGet.cs

```diff
@@ -34,32 +34,32 @@ protected override IDisposable AllocateContextForQueryOperation(out QueryOperati
 
     protected override RavenConfiguration Configuration => RequestHandler.Database.Configuration;
 
-    protected override async ValueTask<FacetedQueryResult> GetFacetedQueryResultAsync(IndexQueryServerSide query, QueryOperationContext queryContext,
+    protected override Task<FacetedQueryResult> GetFacetedQueryResultAsync(IndexQueryServerSide query, QueryOperationContext queryContext,
```

Comment:
Why?

Reply:
The QueryRunner is a Task, previously we had a pattern `return await QueryRunner.Method()` to match the VT type. This way, we can just return actual task without additional async method.